### PR TITLE
Fixed logcat on Android 5.0 devices

### PR DESCRIPTION
**Pull request by todi1856**
### Purpose of PR
**Type of change:**
- [X] Bug fix
- [ ] New feature
- [ ] Code refactor
- [ ] Breaking change
- [ ] Documentation update
- [ ] Other (please describe) 

* **Description of the feature**
While fixing https://fogbugz.unity3d.com/f/cases/1293292/, found a bunch of other bugs related to Android 5.0
- Don't pass --regex to Android 5.0 logcat, since we do the filtering ourselves, this was causing adb logcat to not even spawn
- Process name for /system/bin/netd has been incorrectly resolved, apparently adb.exe shell ps -p 279 -o NAME doesn't work, as in there's no -o flag on Android 5.0 devices, so instead, don't pass -o and simply take the last split of the output which is the process name, backed up by tests now
- Android 5.0 was sending empty lines in some cases, thus spamming the logcat window, we now ignore empty lines again.
- Small improvement, there's often a case where no messages are displayed in logcat, only because you have a tag or filter selected from previous run, in status bar you can now see by what the logcat is filtering, which can hint you why there's no messages 
InfoFiltering

* **Links to screenshots, design docs, user docs, etc.**
* **Links to Jira/Fogbugz cases**
* **Halo effect**
_What areas do you know/expect this feature to affect? Do you have any worries about possible breaking points? Write them down here._
* **Performance Impact**
_Could this PR affect performance? If so, what has been done to measure time taken per frame, memory used, etc.?_
* **Package requirements**
_What Unity Editor version is required/supported? Any other dependancy packages needed?_

### Checklist for PR maker
- [ ] Have you added a backport label? (if needed)
- [X] Have you updated the Changelog? _Each package has a `CHANGELOG.md` file_
- [ ] **Have you added or updated the Documentation to your PR?** 
None
---
### Testing status
* **Existing or new automation tests - what automation was added, changed**
Added new tests for resolving process name, expanded existing test for filtering .
* **Detailed description of what testing was done, what projects were run**
Tested logcat on Android 5.0 LG and Huawei Android 9.0:
- Tried text filtering, regex on/off
- Tried pid filtering for kernel processes, packages
* **What environment did you test on?**
Windows
* **Projects**
Used project in repo
* **Testing checklist**
- [ ] Built and run editor Locally or Katana
- [ ] Built a player Android/iOS (if applicable)
- [ ] Run on device Android/iOS (if applicable)
- [ ] If new feature has UI, does it match Unity style? (Unity [HIG](https://unitytech.github.io/unityeditor-hig/)?)
- [ ] Tested Undo/Redo + Prefab overrides + Alignment in Preset
- [ ] All items have tooltips?
---
### Note to guardian
_Changeset to use, if extra changesets on top. Any other notes to pass along_

### Comments to reviewers
Logcat barely worked on Android 5.0, there was a lot of stuff not working properly, filtering by text, filtering by pid. I checked it, and now it seems to work, but please double check on your side as well

### CODE REVIEW

**Review by JuozasK — APPROVED** [review]
Bug where all tags would be shown, not just the selected ones was fixed, couldn't find other issues. Looks good

File: com.unity.mobile.android-logcat/CHANGELOG.md [hunk 1]
```diff
@@ -10,7 +10,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
  - Removed limitation where Android Logcat could only be used while active Editor platform is Android. Note: Android Support is still required to be installed for Android Logcat to work.
  - Fix Memory Window on Android 11. Android 11 started dumping RSS memory, which was previously unexpected by Memory Window.
  - Include RSS memory in Memory window. 
- - Moved ProjectSettings\AndroidLogcatSettings.asset to UserSettings\AndroidLogcatSettings.asset, since this file wasn't meant to be commited.
+ - Moved ProjectSettings\AndroidLogcatSettings.asset to UserSettings\AndroidLogcatSettings.asset, since this file wasn't meant to be commited. - Fix various issues where logcat would incorretly perform search on phone below Android 7.0
```
Comment (vincent-unity) [inline com.unity.mobile.android-logcat/CHANGELOG.md:13]: I assumed there should be a new line...

File: com.unity.mobile.android-logcat/CHANGELOG.md [hunk 2]
```diff
@@ -10,7 +10,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
  - Removed limitation where Android Logcat could only be used while active Editor platform is Android. Note: Android Support is still required to be installed for Android Logcat to work.
  - Fix Memory Window on Android 11. Android 11 started dumping RSS memory, which was previously unexpected by Memory Window.
  - Include RSS memory in Memory window. 
- - Moved ProjectSettings\AndroidLogcatSettings.asset to UserSettings\AndroidLogcatSettings.asset, since this file wasn't meant to be commited.
+ - Moved ProjectSettings\AndroidLogcatSettings.asset to UserSettings\AndroidLogcatSettings.asset, since this file wasn't meant to be commited. - Fix various issues where logcat would incorretly perform search on phone below Android 7.0
+ - Display information in status bar by which data the messages are filtered, this should help with situtions where no messages are displayed logcat, only because a very specific tag or search filter is set.
```
Comment (vincent-unity) [inline com.unity.mobile.android-logcat/CHANGELOG.md:14]: Really like this idea.

**Review by vincent-unity — APPROVED** [review]

File: com.unity.mobile.android-logcat/CHANGELOG.md (hunk 1 above)
Comment (todi1856) [reply com.unity.mobile.android-logcat/CHANGELOG.md:13]: indeed... this always happens to me when merging...

**Pull request merged**